src/actions/quiz.js

```diff
@@ -1,11 +1,20 @@
 import * as constants from '../constants/quiz';
+import { quiz } from '../data/index.json';
```

Comment:
could you use `data.json` instead of `data/index.json`?

Reply:
In the next commit, we will get data from the Firebase cloud. And we will not use this file